javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/matcher/MuzzleGradlePluginUtil.java

```diff
@@ -135,55 +120,37 @@ public static void assertInstrumentationMuzzled(
   }
 
   /**
-   * Prints all references from all instrumenters present in the passed {@code
+   * Prints all references from all instrumentation modules present in the passed {@code
    * instrumentationClassLoader}.
    *
    * <p>Called by the {@code printMuzzleReferences} gradle task.
    */
   public static void printMuzzleReferences(ClassLoader instrumentationClassLoader) {
-    for (Object instrumenter : loadAllInstrumenters(instrumentationClassLoader)) {
-      if (instrumenter instanceof Instrumenter.Default
-          || instrumenter instanceof InstrumentationModule) {
+    for (InstrumentationModule instrumentationModule :
+        ServiceLoader.load(InstrumentationModule.class, instrumentationClassLoader)) {
+      try {
+        Method getMuzzleMethod =
+            instrumentationModule.getClass().getDeclaredMethod("getMuzzleReferenceMatcher");
+        ReferenceMatcher muzzle;
         try {
-          Method getMuzzleMethod =
-              instrumenter.getClass().getDeclaredMethod("getMuzzleReferenceMatcher");
-          ReferenceMatcher muzzle;
-          try {
-            getMuzzleMethod.setAccessible(true);
-            muzzle = (ReferenceMatcher) getMuzzleMethod.invoke(instrumenter);
-          } finally {
-            getMuzzleMethod.setAccessible(false);
-          }
-          System.out.println(instrumenter.getClass().getName());
-          for (Reference ref : muzzle.getReferences()) {
-            System.out.println(prettyPrint("  ", ref));
-          }
-        } catch (Exception e) {
-          System.out.println(
-              "Unexpected exception printing references for " + instrumenter.getClass().getName());
-          throw new RuntimeException(e);
+          getMuzzleMethod.setAccessible(true);
+          muzzle = (ReferenceMatcher) getMuzzleMethod.invoke(instrumentationModule);
+        } finally {
+          getMuzzleMethod.setAccessible(false);
+        }
+        System.out.println(instrumentationModule.getClass().getName());
+        for (Reference ref : muzzle.getReferences()) {
+          System.out.println(prettyPrint("  ", ref));
         }
-      } else {
-        throw new RuntimeException(
-            "class "
-                + instrumenter.getClass().getName()
-                + " is not a default instrumenter. No refs to print.");
+      } catch (Exception e) {
+        System.out.println(
+            "Unexpected exception printing references for "
+                + instrumentationModule.getClass().getName());
+        throw new RuntimeException(e);
```

Comment:
include message in the re-thrown exception instead of sysout?

Reply:
I'll do both - as I mentioned in another comment, this class is supposed to print out what it's doing since it's a part of the Gradle plugin.